integrations/dell_idrac/README.md

```diff
@@ -0,0 +1,98 @@
+# User Guide | Keeper Security / Dell iDRAC
+
+## Overview
+This user guide covers the post-rotation script for the Keeper Security / Dell iDRAC integration. Details on how to use the post-rotation script are available at the [_Keeper Security online documentation_](https://github.com/Keeper-Security/discovery-and-rotation-saas-dev) and will not be repeated here.
+
+## Dell iDRAC
+The Integrated Dell Remote Access Controller (iDRAC) is designed for secure local and remote server management and helps IT administrators deploy, update and monitor PowerEdge servers anywhere, anytime.
+
+## Pre-requisites
+In order to use the post-rotation script, you will need the following prerequisites:
+
+**1. Requests Library:** Ensure that the requests library is installed in your Python environment. This library is necessary for making HTTP requests to Cisco devices.
+
+**2. Requests library installation:** The Requests library allows you to send HTTP requests easily. Activate a Python virtual environment in your Keeper Gateway environment and install the library using the following command:
+
+    pip install requests
+
+## User setup in Dell iDRAC using sandbox
+### 1. Login to Dell iDRAC Sandbox: 
+- Go to [Dell Open Manage idrac](https://www.dell.com/en-in/lp/dt/open-manage-idrac) 
+- Click on -> **Launch Hands-On Lab** 
+- Launch the sandbox instance.
+- Open Google Chrome which have instance of **dell idrac** and **tester api** instance.
```

Comment:
Is it only supported by Google Chrome or any browser would work? If it works, let's mention Any Web Browser.